src/main/riddl/Order/Order.riddl

```diff
@@ -141,13 +140,13 @@ context OrderContext is {
             and "yield OrderStatusUpdated event"
           }
           example InProcessOrderIllegalStateTransition is {
-            given "order is transitioning from InProcess to Draft, Pending, Cancelled, or Delivered"
+            given "order is transitioning from InProcess to InProcess, Draft, Pending, Cancelled, or Delivered"
```

Comment:
Can somebody cancel an InProcess order? I think they should be able to. I think the rest is valid.

Reply:
I think when Alex and I were talking, it was that an InProcess order would be very hard to cancel. So for that reason, we agreed that an InProcess order can't cancel

Reply:
@improving-app/developers - Generally it is a bad idea to make inconsistencies for the user that are state dependent. They should be able to cancel and order at any time. If the order is "In Process" then the system needs to do what it needs to do to cancel that order, even if it is hard, like recalling an order that is being delivered. In our scenario, there isn't a huge delay between OrderReady and OrderDelivered because of the locality of events to customers.  However, delivery persons could carry a cell phone that would get an instant message to return the item to the store, or other strategies.

Reply:
👍 Will go ahead and allow to cancel and in process order. Will also add a separate ticket for reflecting thing back to the improving-app repository

Reply:
Updated PR to allow InProcess -> Cancelled